tests/functional/spec/list_collector/list_collector_section_summary.spec.js

```diff
@@ -351,9 +356,5 @@ const answerUkBasedQuestion = async () => {
 };
 
 const companiesListRowItem = (row, index) => {
```

Comment:
Index no longer being used here.
```suggestion
const companiesListRowItem = (row) => {
```